plugins/Jobs.cpp

```diff
@@ -280,36 +295,52 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
     }
 
     // ----------------------------------------------------------------------
-    else if (SIEquals(request.methodLine, "GetJob")) {
-        // If we're here it's because peekCommand found some data;
-        // re-execute the query for real now.
+    else if (SIEquals(request.methodLine, "GetJob") || SIEquals(request.methodLine, "GetJobs")) {
+        // If we're here it's because peekCommand found some data; re-execute
+        // the query for real now.  However, this time we will order by
+        // priority.  We do this as three separate queries so we only have one
+        // unbounded column in each query.  Additionally, we wrap each inner
+        // query in a "SELECT *" such that we can have an "ORDER BY" and
```

Comment:
The `ORDER BY` needs to come *after* the `UNION ALL`, which is why I wrapped each of the internal queries in a subselect and then unioned them together.  For example:

```
sqlite> create table blah ( value );
sqlite> select * from blah UNION ALL select * from blah;
sqlite> 
sqlite> select * from blah UNION ALL select * from blah order by value desc;
sqlite> 
sqlite> select * from blah order by value desc UNION ALL select * from blah;
Error: ORDER BY clause should come after UNION ALL not before
sqlite>
```